book-3-workforce/chapters/VIEWMODELS.md

```diff
@@ -51,33 +51,88 @@ namespace StudentExercises.Models.ViewModels
 {
     public class StudentInstructorViewModel
     {
+
         public IEnumerable<Student> Students { get; set; }
         public IEnumerable<Instructor> Instructors { get; set; }
 
-        /*
-            Need to pass along the IConfiguration to get our
-            database connection string
-         */
-        public StudentInstructorViewModel(IConfiguration config)
+        private SqlConnection Connection
+        {
+            get
+            {
+                string _connectionString = "___YOUR CONNNECTION STRING HERE____";
```

Comment:
I think we should pass in the connection string as a constructor parameter.

Reply:
Changed